src/Telemetry/AzureDevOpsTimeline/AzureDevOpsTimeline.cs

```diff
@@ -412,5 +542,26 @@ public AugmentedTimelineIssue(int buildId, string recordId, int index, Issue raw
             public string AugmentedIndex { get; set; }
             public string Bucket { get; set; }
         }
+
+        private class BuildPath
+        {
+            public BuildPath()
+            {
+                BuildId = 0;
+                Path = "";
+                Time = 0;
+            }
+
+            public BuildPath(int buildId, string buildPath, double buildPathTime)
+            {
+                BuildId = buildId;
+                Path = buildPath;
+                Time = buildPathTime;
+            }
+
+            public int BuildId { get; set; }
+            public string Path { get; set; }
+            public double Time { get; set; }
```

Comment:
The fact this string is a special encoding of a list is... strange. Why not have it be an actual list?